src/common/target.c

```diff
@@ -168,7 +168,6 @@ static const TargetEntry TargetMap[] = {
     {   "sim6502",      TGT_SIM6502     },
     {   "sim65c02",     TGT_SIM65C02    },
     {   "supervision",  TGT_SUPERVISION },
-    {   "vc20",         TGT_VIC20       },
```

Comment:
its only called vc20 in germany, and vic20 everywhere else... so why would it be vc20? :)